README.md

```
@@ -38,9 +38,9 @@ Launch the application in dev mode, if using the ubnutu example vagrant vm:
 bundle exec rails server -b 0.0.0.0
 ```
 
-(optional) in another session to speed up any webpack related development
+(optional) in another session, to rebuild JS automatically as you edit `app/javascript`
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)